Made changes to page by adding htc explanation, comparison table, and summary of htc origin (#1000)
CHTC/chtc-website-source: htc.md

title: What is High Throughput Computing?
---

In computing, **throughput** is a measure of the number of computing tasks a system can complete over time. 

> **High Throughput Computing (HTC)** is an approach to computing that focuses on completing as much work as possible by focusing on running multiple independent tasks in parallel across available computers.

Reviewer: ```suggestion
**High Throughput Computing (HTC)** is an approach to computing that focuses on completing as much work as possible by running **multiple independent tasks** in across **multiple computers** as they become available.
```

> ### What kinds of work are good for HTC?

Reviewer: ```suggestion
### What kinds of computational works well with the HTC approach?
```
